logger.go

```diff
@@ -242,7 +242,7 @@ func LoggerWithConfig(conf LoggerConfig) HandlerFunc {
 	return func(c *Context) {
 		// Start timer
 		start := time.Now()
-		path := c.Request.URL.Path
+		path := c.Request.URL.EscapedPath()
```

Comment:
![image](https://github.com/user-attachments/assets/d5c4baaa-b09e-45ef-8126-75b0e5028de0)

using `Path` it seem something wrong in this case.